### ISSUE TITLE
Postinstall script causes issues on packer build

### ISSUE THREAD

**Issue by submitteddenied** [evantahler/ah-sequelize-plugin#7]
For some reason when my packer build script runs `npm rebuild` I get the following error.

```
amazon-ebs: > ah-sequelize-plugin@0.3.2 postinstall /var/www/sites/node-app/node_modules/ah-sequelize-plugin
amazon-ebs: > scripts/postinstall.js
amazon-ebs:
amazon-ebs: coppying /var/www/sites/team-service/node-app/ah-sequelize-plugin/config/sequelize.js to /var/www/sites/node-app/config/sequelize.js
amazon-ebs:
amazon-ebs: events.js:72
amazon-ebs: throw er; // Unhandled 'error' event
amazon-ebs: ^
amazon-ebs: Error: ENOENT, open '/var/www/sites/node-app/config/sequelize.js'
```

I am not able to reproduce this locally, even if I run exactly the same steps in my local environment.

I also have other `ah-xxx-plugins` that have identical `postinstall.js` scripts for copying in base configs (I copied them from `ah-sequelize-plugin`) but they don't cause the same issue.

I thought it might be related to the `mkdirp` call, but the `ENOENT` refers to attempting to open `config/sequelize.js`. Any idea what might be going on here?

**Comment by submitteddenied** [comment]
I tracked down the issue. My build script replaces `config/sequelize.js` with a symlink to `/etc/node-app/sequelize.js` before running the `npm rebuild`. Since this file (ie, `/etc/node-app/sequelize.js`) does not exist until the image is deployed, the postinstall script attempts to copy the template from the `ah-sequelize-plugin` directory. This fails either because the target of the link is in `/etc` and it doesn't have permission, or because `.pipe()` doesn't work into symlinks.

I'll create a PR that addresses this by also checking to see if `config/sequelize.js` is a symlink before attempting to write there - this should indicate that the user is handling that config file, so the postinstall should not try to write it.

**Comment by evantahler** [comment]
perfect, thanks!

On a related note, how would you feel about removing the post install?  It's nice to have that file "appear" when you install the package, but I don't think we should be messing with files in production... folks can copy the example if they need...

**Comment by submitteddenied** [comment]
I was thinking about that too. I think a well-documented example (in this case, probably linking to the Sequelize docs) would do the trick.

We could also validate that the required configuration options are present in the initializer (and fail to start actionhero if they're missing!)...

**Comment by submitteddenied** [comment]
Fixed with PR #8.

**Comment by evantahler** [comment]
Cool.  `lstat` fixes the current postinstall script, but I think I'd rather remove the postinstall all together.
